.circleci/config.yml

```diff
@@ -114,6 +117,28 @@ jobs:
           upload_name: Coverage Report for iOS Tests
           xtra_args: -c -v --xc --xp iosresults.xcresult
   
+  test-ios-integration:
+    macos:
+      xcode: 13.4.0 # Specify the Xcode version to use
+
+    steps:
+      - checkout
+
+      - install_dependencies
+
+      - prestart_ios_simulator
+
+      - run:
+          name: Run iOS Tests
+          command: make test-integration-upstream
+
+      # Code coverage upload using Codecov
+      # See options explanation here: https://docs.codecov.com/docs/codecov-uploader
+      - codecov/upload:
```

Comment:
Do we need to upload code coverage for integration? I don't think we need to get code coverage for these tests since we are not going to run these with every build.